platform/discord/manifest.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: discord-webhook-service
+spec:
+  template:
+    spec:
+      containers:
+        - image: pocketsizefund/discord:latest
+          ports:
+            - containerPort: 8080
+          env:
+            - name: DISCORD_WEBHOOK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: platform
+                  key: discord_webhook_url
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: discord-webhook-trigger
```

Comment:
Is "webhook" the right terminology? Genuinely curious so that we can be consistent.

Reply:
yea, we could call it `bot` but since it's really only a 1-way communication right now I didn't want to imply it was interactive like a bot and is just a fire-and-forget messenger, but we could just rename it `discord` for future use as a possible bot

Reply:
I changed this to just `discord`